.circleci/config.yml

```diff
@@ -184,7 +184,7 @@ jobs:
   test_browser:
     <<: *defaults
     docker:
-      - image: mcr.microsoft.com/playwright@sha256:c3c8a833254aaad31b2d336e67b0dbb9af744a12ab486e18f230033b99649c4f
+      - image: mcr.microsoft.com/playwright:v1.24.2-focal
```

Comment:
Version tag is more readable.
I've used https://mcr.microsoft.com/en-us/product/playwright/tags to determine whether we were using `-focal` version or not